starter-code/src/clue.js

```diff
@@ -85,10 +85,66 @@ name: Patio
 */
 
 // Characters Collection
-var charactersArray = [];
+var charactersArray = [
+    { First_name :"Jacob"  ,last_name:"Green"   ,color:"Green" , description:"He has a lot of connections"  ,age: 45,occupation:"Entrepeneur" },
+    { First_name : "Doctor"   ,last_name: "Orchid"  ,color:"White"     , description:"PhD in plant toxicology. Adopted daughter of Mr. Boddy"  ,age: 26 ,occupation:"Scientist"  },
+    { First_name :"Victor"    ,last_name:  "Plum" ,color: "purple"    , description: "Billionare video game designer" ,age: 22,occupation: "Designer" },
+    { First_name :  "Kasandra"  ,last_name: " Scarlet"  ,color: "red"    , description:" She is an A-list movie star with a dark past"  ,age: 31,occupation:  "Actor"},
+    { First_name :"Eleanor"  ,last_name:"Peacock"   ,color: "blue"    , description: "She is from a wealthy family and uses her status and money to earn popularity" ,age: 36 ,occupation: "Socialité"  },
+    { First_name : "Jack"   ,last_name:"Mustard"   ,color:"yellow"     , description: " He is a former football player who tries to get by on his former glory" ,age: 62,occupation: "Retired Football player" },
+];
 
 // Rooms' Collection
-var roomsArray = [];
+var roomsArray = [
+   {name: "Dinning Room"},
+   { name: "Conservatory"},
+   { name: "Kitchen"},
+   { name:" Study"},
+   { name: "Library"},
+   {name: "Billiard Room"},
+   {name: "Lounge"},
+   {name:" Ballroom"},
+   { name: "Hall"},
+   { name: "Spa"},
+   { name: "Living Room"},
+   {name: "Observatory"},
+   {name: 'Theater'},
+   { name: "Guest House"},
+   { name: "Patio"},
+];
 
 // Weapons Collection
-var weaponsArray = [];
+var weaponsArray = [
+    {name:"rope", weight:10},
+    {name:"knife", weight:8},
+    {name:"candlestick", weight:2},
+    {name:"dumbbell", weight:30},
+    {name:"poison", weight:2},
+    {name:"axe", weight:15},
+    {name:"bat", weight:13},
+    {name:"trophy", weight:25},
+    {name:"pistol", weight:20},
+];
+
+
+function randomSelector(array) {
+	return array[Math.floor(Math.random() * array.length)];
+}
+
+function randomSelector(array) {
+	return array[Math.floor(Math.random() * array.length)];
+}
+
+function pickMistery() {
+	var misteryEnvelope = [];
+	misteryEnvelope.push(randomSelector(charactersArray));
+	misteryEnvelope.push(randomSelector(weaponsArray));
+	misteryEnvelope.push(randomSelector(roomsArray));
+	return misteryEnvelope;
+}
+var misteryEnvelope = pickMistery();
+
+function revealMistery() {
```

Comment:
function revealMistery(mistery) {
    return mistery[0].first_name + ' ' + mistery[0].last_name + ' killed Mr.Boddy using the ' + mistery[1].name + ' in the ' + mistery[2].name + '!!!!';
  }
  
  var misteryEnvelope = pickMistery();
  console.log(revealMistery(misteryEnvelope));